### DISCUSSION TITLE
Borgbackup exclude image previews

### DISCUSSION THREAD

**Discussion by Handkep** [nextcloud/all-in-one#5433] [Ideas] [answered]
As the title says, it would be nice if one could decide to packup all the preview data or not. One could also allow to exclude any other folder to only back up things that are important because preview files can be regenerated and thus only take up space.

**Comment by szaimen** [comment]
Hi, I agree that restoring the backup should offer such an option to exclude previews.

**Comment by szaimen** [comment, marked as answer]
I created https://github.com/nextcloud/all-in-one/issues/5451 to track this
